devtools.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency 'rspec',        '~> 3.4.0'
   gem.add_runtime_dependency 'rspec-core',   '~> 3.4.0'
   gem.add_runtime_dependency 'rspec-its',    '~> 1.2.0'
-  gem.add_runtime_dependency 'rake',         '~> 10.4.2'
+  gem.add_runtime_dependency 'rake',         '~> 11.1.x'
```

Comment:
If you say `~> 11.1.1` it'll allow `~> 11.1.3` but not `~> 1.11.0`. AFAIK `~> 1.11.x` will allow `~> 1.11.0`.